lib/themes/dosomething/paraneue_dosomething/js/components/InstructionSlide.js

```diff
@@ -14,7 +14,7 @@ class InstructionSlide extends React.Component {
 
   render() {
     return (
-      <div className={`slideshow__slide ${this.props.isActive ? '-active -' + this.props.animationDirection : ''}`}>
+      <div className={`slideshow__slide ${this.props.isActive ? '-active fade-in': ''}`}>
```

Comment:
Ironically, this is something `ReactCSSTransitionGroup` would handle perfectly! 😆